ui/apps/platform/src/services/PoliciesService.ts

```diff
@@ -7,22 +6,11 @@ import { addBrandedTimestampToString } from 'utils/dateUtils';
 import { transformPolicyCriteriaValuesToStrings } from 'utils/policyUtils';
 
 import axios from './instance';
-import { policy as policySchema } from './schemas';
 
 const baseUrl = '/v1/policies';
 const policyCategoriesUrl = '/v1/policyCategories';
 
-/*
- * Fetch policy summary for a given policy policyId.
- * TODO delete after policiesSagas.js has been deleted, because superseded by getPolicy
- */
-export function fetchPolicy(
-    policyId: string
-): Promise<{ response: { entities: { policy: Record<string, Policy> } } }> {
-    return axios.get<Policy>(`${baseUrl}/${policyId}`).then((response) => ({
-        response: normalize(response.data, policySchema),
-    }));
-}
+type Empty = Record<string, never>;
```

Comment:
should we pull this out into a separate utils for service return types? not sure if it is currently used anywhere else but it doesn't seem like it's policy specific enough to stay here.

Reply:
Good eyes. I will merge and we can brainstorm about pro and con of local or shared type definition.

For now, `Empty` is an experimental idiom with two goals:
* Mimic `Empty` which seems to be built into proto notation.
* Reduce prettier wrapping.

There is no `export` because callers will never import it. It tells TypeScript to report an error if a caller destructures the response (for example, a PUT method which returns empty response instead of updated object).

By the way, `.then((response) => response.data)` method call is important even (or especially) for empty responses, otherwise the function leaks the sibling properties of the response object as if they were data!